src/ussd_registration.js

```diff
@@ -6,6 +6,7 @@ go.app = (function() {
   var moment = require("moment");
   var Choice = vumigo.states.Choice;
   var ChoiceState = vumigo.states.ChoiceState;
+  //var PaginatedState = vumigo.states.PaginatedState;
```

Comment:
Could we just remove this line, instead of commenting it out?

Reply:
It shouldn't be there. I'll remove it